lib/BaseStripeClient.php

```diff
@@ -29,21 +29,42 @@ class BaseStripeClient implements StripeClientInterface
     private $filesBase;
 
     /**
-     * Initializes a new instance of the {@link StripeClient} class.
+     * Initializes a new instance of the {@link BaseStripeClient} class.
      *
-     * @param null|string $apiKey the API key used by the client to make requests
-     * @param null|string $clientId the client ID used by the client in OAuth requests
-     * @param null|string $apiBase The base URL for Stripe's API. Defaults to {@link DEFAULT_API_BASE}.
-     * @param null|string $connectBase The base URL for Stripe's OAuth API. Defaults to {@link DEFAULT_CONNECT_BASE}.
-     * @param null|string $filesBase The base URL for Stripe's Files API. Defaults to {@link DEFAULT_FILES_BASE}.
+     * The constructor takes a single argument. The argument can be a string, in which case it
+     * should be the API key. It can also be an array with various configuration settings.
+     *
+     * Configuration settings include the following options:
+     *
+     * - api_key (string): the Stripe API key, to be used in regular API requests.
+     * - client_id (string): the Stripe client ID, to be used in OAuth requests.
+     * - api_base (string): the base URL for regular API requests. Defaults to
+     *   {@link DEFAULT_API_BASE}. Changing this should rarely be necessary.
+     * - connect_base (string): the base URL for OAuth requests. Defaults to
+     *   {@link DEFAULT_CONNECT_BASE}. Changing this should rarely be necessary.
+     * - files_base (string): the base URL for file creation requests. Defaults to
+     *   {@link DEFAULT_FILES_BASE}. Changing this should rarely be necessary.
```

Comment:
you're missing `stripe_account` right?

Reply:
I haven't added any new configuration options yet, like `stripe_version` or `stripe_account`. I'll do that in a followup PR.